specs/trip_dispatch_spec.rb

```diff
@@ -111,4 +95,67 @@
       expect(passenger.trips).must_include trip
     end
   end
+
+  describe '#request_trip method' do
+    before do
+      @requested_trip = @dispatcher.request_trip(1)
+    end
+
+    it "generates an array of :AVAILABLE drivers" do
+      available_drivers = @dispatcher.generate_available_drivers
+      expect(available_drivers).must_be_kind_of Array
+      expect(available_drivers.first.status).must_equal :AVAILABLE
+      expect(available_drivers.last.status).must_equal :AVAILABLE
+    end
+
+    it "returns an array of :AVAILABLE drivers that aren't the Passenger" do
+      available_drivers = @dispatcher.check_drivers_not_passenger?(2)
+      expect(available_drivers).must_be_kind_of Array
+
+      available_drivers.each do |driver|
+        expect(driver.id).wont_equal 2
+      end
+    end
+
+    it "Assigns the first :AVAILABLE driver with no driven trips to a new Trip" do
+      driver  = @requested_trip.driver
+      expect(driver).must_be_instance_of RideShare::Driver
+      expect(driver.driven_trips.length).must_equal 1
```

Comment:
None of these tests answer the question: what do these helper methods do if no suitable driver is available?